justfile

```diff
@@ -0,0 +1,8 @@
+default:
+    just --list
+
+test:
+    python -m unittest sad_forms.unit_tests.validator_environment_variables
```

Comment:
we need to run all files, not only 1 file -- check history of shell commands, I've found solution for that